test/e2e/lib/driver-helper.js

```diff
@@ -512,6 +512,15 @@ export async function scrollToBottom( driver, selector ) {
 	);
 }
 
+export async function scrollToTop( driver, selector ) {
```

Comment:
Now we have `scrollIntoView`, `scrollToBottom` and `scrollToTop` which basically do similar thing. I'll review and make one function for all of them in a separate PR.